stdlibs/std/banker.go

```diff
@@ -15,7 +15,7 @@ package std
 // type, and those can't return non-primitive objects
 // (without confusion).
 type Banker interface {
-	GetCoins(addr Address, dst *Coins)
+	GetCoins(addr Address) (dst Coins)
```

Comment:
all to make this possible with native pkgs/std.Coins -> stdlibs/std.Coins mapping.